monitor/routerecon/routescan_internal_test.go

```diff
@@ -15,9 +15,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-//go:generate go test . -integration -v -run=TestQueryLatestXChain
+var (
+	integration     = flag.Bool("integration", false, "run routescan integration tests")
+	routeScanAPIKey = flag.String("routeScanAPIKey", "", "RouteScan API key for enabling increased rate limiting")
```

Comment:
no need to prefix stuff in routescan package with routescan, that is called "stuttering" in golang.

Reply:
But the name of the flag I think should remain `routeScanAPIKey`. For the var name I agree.